Projects/TMProject/TMGlobal.h

```diff
@@ -228,7 +228,12 @@ const static stTeleportPos g_TeleportTable[37] =
   { 3648, 3140, 700, g_pMessageStringTable[211] },
   { 2480, 1648, 700, g_pMessageStringTable[301] },
   { 1052, 1708, 0, g_pMessageStringTable[321] },
-  { 1056, 1724, 0, g_pMessageStringTable[485] }
+  { 1056, 1724, 0, g_pMessageStringTable[485] },
```

Comment:
Fiquei na dúvida quanto as posições. 
Você seguiu o padrão das posições acima? Se não me engano, a posição fica centralizada no teleporte. Gostaria apenas de confirmar

Reply:
Segui conforme o padrão dos outros teleports, com exceção do portal da sala do kefra.

![](https://i.imgur.com/rzOBBTc.png)
![](https://i.imgur.com/rvDwbXd.png)
![](https://i.imgur.com/X6WineU.png)

Reply:
eu fiz como solicitado , o teleporte de Noatun -> Khepra foi corrigido . Mas o teleporte de Khepra -> Noatun ainda continuas bugado mesmo eu colocando a coordenada da forma que vc pôs na imagem 2364, 4072

Reply:
Bom, eu vou dar uma olhada nas coordenadas e analisar o problema melhor antes de mergear.

De qualquer forma, eu solicite mudanças quanto a array. Quando tiver um tempo para isso :)

Reply:
opa meu querido acabei de notar que tem que alterar aqui tb 
em : TMFieldScene.cpp
na linha 7860 :  for (int ll = 0; ll < 42; ++ll)